tutorials/wordle.md

```diff
@@ -187,15 +187,15 @@ Run the following command inside the `wordle` directory.
 ::: code-group
 
 ```bash [local-celestia-devnet]
-go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.46.13-rollkit-v0.9.0-no-fraud-proofs
-go mod edit -replace github.com/tendermint/tendermint=github.com/rollkit/cometbft@v0.0.0-20230524013049-75272ebaee38
+go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.47.3-rollkit-v0.10.1-no-fraud-proofs
+go mod edit -replace github.com/cometbft/cometbft=github.com/rollkit/cometbft@v0.0.0-20230524013049-75272ebaee38
 go mod tidy
 go mod download
 ```
 
 ```bash [Arabica Devnet]
-go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.46.13-rollkit-v0.9.0-no-fraud-proofs
-go mod edit -replace github.com/tendermint/tendermint=github.com/rollkit/cometbft@v0.0.0-20230524013049-75272ebaee38
+go mod edit -replace github.com/cosmos/cosmos-sdk=github.com/rollkit/cosmos-sdk@v0.47.3-rollkit-v0.10.1-no-fraud-proofs
+go mod edit -replace github.com/cometbft/cometbft=github.com/rollkit/cometbft@v0.0.0-20230524013049-75272ebaee38
```

Comment:
No need for cometbft replace, as we don't rely on our fork anymore

Reply:
meaning it can be taken out?

![nice](https://media0.giphy.com/media/WXLScM6bGOgGwJ8SXH/200w.webp)

Reply:
yes